openc3/lib/openc3/models/gem_model.rb

```diff
@@ -17,50 +17,41 @@
 # All changes Copyright 2022, OpenC3, Inc.
 # All Rights Reserved
 
+require 'fileutils'
 require 'open-uri'
 require 'nokogiri'
 require 'httpclient'
 require 'rubygems'
 require 'rubygems/uninstaller'
 require 'tempfile'
-require 'openc3/utilities/bucket'
 require 'openc3/utilities/process_manager'
-require 'openc3/api/api'
+require "pathname"
 
 module OpenC3
   # This class acts like a Model but doesn't inherit from Model because it doesn't
   # actual interact with the Store (Redis). Instead we implement names, get, put
   # and destroy to allow interaction with gem files from the PluginModel and
   # the GemsController.
   class GemModel
-    extend Api
-
     def self.names
-      bucket = Bucket.getClient()
-      gems = []
-      bucket.list_objects(bucket: ENV['OPENC3_GEMS_BUCKET']).each do |object|
-        gems << object.key
-      end
-      gems
+      result = Pathname.new("#{ENV['GEM_HOME']}/gems").children.select { |c| c.directory? }.collect { |p| File.basename(p) + '.gem' }
```

Comment:
I thought it was #{ENV['GEM_HOME']}/cache ... not /gems

Reply:
cache is where the .gem files are stored.   /gems/gems is the folders for gems that are actually installed.